pkg/ccl/backupccl/file_sst_sink.go

```diff
@@ -264,6 +267,21 @@ func (s *fileSSTSink) write(ctx context.Context, resp exportedSpan) (roachpb.Key
 		return nil, err
 	}
 
+	// extend determines if the new span should be added to the last span. This
+	// must occur if the previous span ended mid row, or if the new span is a
```

Comment:
hype-nit: `must` here could be read as applying to either side of the `or`, I might add a `may` to the latter half to make it clearer ti is optional?

Reply:
changed must to will